### PR TITLE
Adding performance instrumentation to the SDK

### PR DESCRIPTION
**Pull request by deanhaber**

### CODE REVIEW

File: lib/nlu/factory.js [hunk 1]
```diff
@@ -122,7 +125,8 @@ module.exports = Factory;
 
 function getNLU(type, name) {
     return new Promise(function (resolve, reject) {
-            const nlu = require(`../../../../res/nlu/${type}`);
+        console.log(appDir);
```
Comment (offerakrabi) [inline lib/nlu/factory.js]: remove log

File: res/nlu/wcs.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
 {
   "workspace": {
     "en-US": {
-      "name": "your workspace name",
-      "workspace_id": "your workspace id"
+      "name": "dean",
```
Comment (offerakrabi) [inline res/nlu/wcs.json]: set all these back to default values

File: lib/nlu/factory.js (hunk 1 above)
Comment (deanhaber) [reply lib/nlu/factory.js]: Done

File: res/nlu/wcs.json (hunk 2 above)
Comment (deanhaber) [reply res/nlu/wcs.json]: Done

**Pull request closed without merging**